### PR TITLE
Check there is no storagecluster on consumer

### PR DESCRIPTION
**Pull request by ebondare**

### CODE REVIEW

File: ocs_ci/helpers/managed_services.py [hunk 1]
```diff
@@ -581,6 +582,14 @@ def verify_faas_consumer_resources():
     log.info(f"Client endpoint IP is {ep_ip}")
     assert re.match("\\d+(\\.\\d+){3}", ep_ip)
 
+    # Check that there's no storagecluster on consumer
+    sc_obj = OCP(
+        kind=constants.STORAGECLUSTER,
+        namespace=config.ENV_DATA["cluster_namespace"],
+    )
+    log.info(f"Verifying there's no storagecluster: {sc_obj['items']}")
+    assert sc_obj["items"] == {}
```
Comment (jilju) [inline ocs_ci/helpers/managed_services.py:591]: It will be nice to have an error message.

Comment (jilju) [inline ocs_ci/helpers/managed_services.py:591]: Fetch the storagecluster resources before verifying.

Comment (ebondare) [reply ocs_ci/helpers/managed_services.py:591]: Which resources do you like me to fetch?

Comment (ebondare) [reply ocs_ci/helpers/managed_services.py]: thank you, fixed

Comment (jilju) [reply ocs_ci/helpers/managed_services.py]: This is fixed in the recent commit.
```
sc_obj.**get()**
```

**Review by jilju — APPROVED** [review]

**Review by yitzhak12 — DISMISSED** [review]

**Comment by ebondare** [comment]
Successful test run: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/26001/consoleFull

**Review by fbalak — APPROVED** [review]

**Review by yitzhak12 — APPROVED** [review]

**Pull request merged**